Grbl_Esp32/Grbl_Esp32.ino

```diff
@@ -20,8 +20,29 @@
 
 #include "src/Grbl.h"
 
+class CustomGCmdM102:public CustomGCmd
+{
+  public:
+  CustomGCmdM102():CustomGCmd('M',102)
+  {
```

Comment:
Please use the same formatting style as the rest of the code.  There is a .clang-format file at the top level that will do the formatting for you in conjunction with the widely-supported clang-format tool.

Reply:
OK will do